libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -382,13 +386,17 @@ class AP_InertialSensor : AP_AccelCal_Client
         bool Write_ISBH(const float sample_rate_hz) const;
         bool Write_ISBD() const;
 
+        bool has_option(batch_opt_t option) const { return _batch_options_mask & uint16_t(option); }
+
         uint64_t measurement_started_us;
 
         bool initialised : 1;
         bool isbh_sent : 1;
         bool _doing_sensor_rate_logging : 1;
         bool _doing_post_filter_logging : 1;
+        bool _doing_pre_post_filter_logging : 1;
         uint8_t instance : 3; // instance we are sending data for
+        bool post_filter : 1; // whether we are sending post-filter data
```

Comment:
we should remove these as bitfields, just make bool, we did this to save ram, but it costs flash, not worth it

Reply:
ok

Reply:
Fixed